components/Seo.php

```diff
@@ -100,22 +106,32 @@ private function getDataFromStaticPage()
             // If RainLab.Translate plugin is installed we need to ensure, that
             // lang prefix is not in our way, so we are removing it from url.
             if (PluginManager::instance()->hasPlugin('RainLab.Translate')) {
-                if (Str::contains($url, '/')) {
-                    $url = explode('/', $url)[1];
-                }
+                // Remove language prefix from URL to get clean URL
+                // Simply exploding the URL after first slash does not work for 
+                // multilevel websites
+                $url = substr($url, 3); 
+
+                // If we're in root then use root as URL
+                if (!strlen($url))
+                    $url = '/';
```

Comment:
```suggestion
                if (!strlen($url)) {
                    $url = '/';
                }
```

Please don't omit braces. It tends to create errors or problems while debugging sometimes.